scripts/scenarios/cloud6/cloud7-upgrade-non-disruptive-ceph.yml

```diff
@@ -93,6 +93,7 @@ proposals:
         admin_keyring: "/etc/ceph/ceph.client.admin.keyring"
         pool: volumes
         user: cinder
+        secret_uuid: 75ffb452-2478-4c4c-abcc-1be39cb6325a
```

Comment:
Where does this value come from?
Does it matter that it will be neither secret nor unique?
Why can't barclamp-cinder or ceph handle this?